compiler/luci/pass/src/ResolveFormerCustomOpPass.test.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/ResolveFormerCustomOpPass.h"
+
+#include <gtest/gtest.h>
+
+TEST(ResolveFormerCustomOpPassTest, name)
+{
+  luci::ResolveFormerCustomOpPass pass;
+  auto const name = pass.name();
+  ASSERT_NE(nullptr, name);
+}
```

Comment:
(in another PR)
It would be nice to have some positive and negative tests for new introduced codes.
I don't see them in the draft. Can you apply them in your draft PR?
1/ unit test codes in this `ResolveFormerCustomOpPass.test.cpp` file
2/ integration test in `luci-pass-value-py-test` module

Reply:
@seanshpark 
> I don't see them in the draft. Can you apply them in your draft PR?

Thank you for your guide! :)
I will apply it in Draft PR including some positive and negative tests.